sdk/assets/model/model.ipynb

```diff
@@ -2,6 +2,7 @@
   "cells": [
```

Comment:
the example 2.4 still uses model_uri. Updated 2.3 in the sdk-preview branch to use file_model.path, to show how you can register from a file location.
The cloud URI and Job URI (2.3 and 2.4) didn't pass the CI test, so i removed hardcoding.. Not sure how that testing works.
I used **type="custom"** for the type in the notebook, why did you have to use type=ModelType.CUSTOM

Reply:
Thanks for updating the paths.
We cannot use hardcoded resource groups, subscriptions or workspaces in the samples. Firstly, that information cannot be shared. Second, it will fail testing. The alternate is to do a run and use that run ID in the same notebook so that we dont need to hardcode it. I have commented out the code to create the model. We have a sample code, but user can substitute with required details and uncomment it.
We dont want users to remember keywords or types. Hence i have used an enum value in the sample.